WalkingChallenge/ProfileViewController.swift

```diff
@@ -65,6 +69,14 @@ class ProfileViewController: UIViewController {
       ConstraintMaker.leading.trailing.equalToSuperview().inset(Style.Padding.p12)
     }
 
+    // TODO(compnerd) make this look better ...
+    rangePicker.layer.borderColor = Style.Colors.grey.cgColor
+    rangePicker.layer.borderWidth = 1
+    rangePicker.snp.makeConstraints { (ConstraintMaker) in
```

Comment:
snapkit convention names the objects to `make` so it reads `make.top.equalTo...`